### PR TITLE
Added instructions for installing Pipelines.

### PR DESCRIPTION
**Pull request by hbelmiro**
Resolves https://github.com/kubeflow/website/issues/3714.

This PR aims to add minimal information to install KFP standalone.
Further improvements will be added later.

### CODE REVIEW

File: content/en/docs/components/pipelines/operator-guides/installation/_index.md [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
\ No newline at end of file
+You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
+[kubectl](https://kubernetes.io/docs/reference/kubectl/overview/), and [kustomize](https://kustomize.io/).
+
+## Deploying Kubeflow Pipelines
+
+1. Deploy the Kubeflow Pipelines:
+
+     ```
+     export PIPELINE_VERSION={{% pipelines/latest-version %}}
```
Comment (diegolovison) [inline content/en/docs/components/pipelines/operator-guides/installation/_index.md:21]: I think we can do it automatically by looking at `https://raw.githubusercontent.com/kubeflow/website/master/layouts/shortcodes/pipelines/latest-version.html`.

Comment (hbelmiro) [reply content/en/docs/components/pipelines/operator-guides/installation/_index.md:21]: `{{% pipelines/latest-version %}}` is replaced with the latest version during the build.
You can see that in: https://deploy-preview-3793--competent-brattain-de2d6d.netlify.app/docs/components/pipelines/operator-guides/installation/

File: content/en/docs/components/pipelines/operator-guides/installation/_index.md [hunk 2]
```diff
@@ -1,11 +1,27 @@
 +++
 title = "Installation"
-description = "Options for deploying Kubeflow Pipelines"
```
Comment (thesuperzapper) [inline content/en/docs/components/pipelines/operator-guides/installation/_index.md:3]: Why was this removed?? All pages must have descriptions.

File: content/en/docs/components/pipelines/operator-guides/installation/_index.md [hunk 3]
```diff
@@ -1,11 +1,27 @@
...
 weight = 1
 +++
 
 {{% kfp-v2-keywords %}}
 
-This page will be available soon. For similar information, see [KFP v1 installation documentation][v1-installation].
+As an alternative to deploying Kubeflow Pipelines (KFP) as part of the
+[Kubeflow deployment](/docs/started/#installing-kubeflow), you also have a choice
```
Comment (thesuperzapper) [inline content/en/docs/components/pipelines/operator-guides/installation/_index.md:9]: What is "Kubeflow deployment", this is not a defined term.

I assume you mean "Kubeflow Platform".

Also, this link is wrong.

File: content/en/docs/components/pipelines/operator-guides/installation/_index.md [hunk 4]
```diff
@@ -1,11 +1,27 @@
...
+to deploy only Kubeflow Pipelines. Follow the instructions below to deploy
+Kubeflow Pipelines standalone using the supplied kustomize manifests.
 
-[v1-installation]: /docs/components/pipelines/legacy-v1/installation/
\ No newline at end of file
+You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
+[kubectl](https://kubernetes.io/docs/reference/kubectl/overview/), and [kustomize](https://kustomize.io/).
+
+## Deploying Kubeflow Pipelines
```
Comment (thesuperzapper) [inline content/en/docs/components/pipelines/operator-guides/installation/_index.md:16]: Please format this more like the ["Model Registry" page](https://www.kubeflow.org/docs/components/model-registry/installation/) with two sections:

1. Prerequisites
1. Installing Kubeflow Pipelines:
     - Installing on Kubeflow Platform
          - Explain what this means, and the differences to using standalone.
          - Core differences to standalone include "multi user support", "connecting to external object stores", "connecting to external MySQL databases", etc.
    - Standalone Installation:
         - The steps you have already listed.
3. Next Steps

**Review by thesuperzapper — COMMENTED** [review]
@rimolive in what universe was this PR ready to merge?

Please do not merge significant changes without actually reviewing the content.

There are multiple glaring issues with this PR, ranging from dead links to a near complete lack of important information for the user.

Put yourself in the shoes of someone trying to install Kubeflow Pipelines, would this page be helpful in its current state?

**Comment by rimolive** [comment]
@thesuperzapper I am removing myself from kfp website approvers. I don't feel I can continue working with approving these PRs given the other responsibilities I already have.

My apologies to this, we'll find another approver.

**Comment by gregsheremeta** [comment]
> in what universe was this PR ready to merge?

Mathew, this language is unacceptable in a professional setting, and in an open source community. It goes against the CNCF Code of Conduct.

It's certainly ok to express disapproval of a PR, but please treat your peers with respect.

File: content/en/docs/components/pipelines/operator-guides/installation/_index.md (hunk 3 above)
Comment (hbelmiro) [reply content/en/docs/components/pipelines/operator-guides/installation/_index.md:9]: > What is "Kubeflow deployment", this is not a defined term.
> 
> I assume you mean "Kubeflow Platform".

As requested in the issue, which BTW is open since April and was planned for Kubeflow 1.9 by the release team, this was copied from https://github.com/kubeflow/website/blob/37c939d436b53836629898ce8be2f7715ab4f8cc/content/en/docs/components/pipelines/legacy-v1/installation/standalone-deployment.md?plain=1#L14
Feel free to send a PR to improve both pages.

File: content/en/docs/components/pipelines/operator-guides/installation/_index.md (hunk 4 above)
Comment (hbelmiro) [reply content/en/docs/components/pipelines/operator-guides/installation/_index.md:16]: I don't have the bandwidth to add instructions for all of this at the moment. I did what I could offer. I agree we need more information on this page. Are you willing to send a PR?

**Comment by chasecadet** [comment]
| in what universe was this PR ready to merge?
@thesuperzapper I think this isn't effective to help get the outcome you need. You have a lot of users and a lot of insight into how they see/use KFP. I work with a lot of engineers and it's that weird "it's now obvious to me" but we forget that our insights had to be ingested and processed deeper into our technical muscle memory. Totally understandable especially as someone looking to move fast and continue to grow their distribution. I can understand the anxiety/claustrophobia there.  I think you do raise some interesting comments/concerns here

>What is "Kubeflow deployment", this is not a defined term. I assume you mean "Kubeflow Platform".

Let's determine how we as a community refer to Kubeflow as a whole and as individual components. 
"A Kubeflow deployment" or "A deployed Kubeflow platform" are two different things because of our permutation situation and lack of conformance. 

> Why was this removed?? All pages must have descriptions.
What might help here is surfacing your concern and the "why" we (if this was a group decisions) require pages to have descriptions and then asking if someone would be willing to adjust it.  People like to provide value vs. following orders and the goal here is high quality documentation.


> Put yourself in the shoes of someone trying to install Kubeflow Pipelines, would this page be helpful in its current state?

The interesting bit here is your empathy for someone installing KFP. I love the user empathy. I think what might help your goal of ensuring quality for the community and users is being less assertive here and unpacking your concerns directly. "I am concerned a user might get confused by XYZ and then lead to frustrations and them abandoning the project" etc.. then we can as a community address those concerns.  You are invested in many ways in the health of Kubeflow and it shows. I believe if you work to surface your experience, expertise, and concerns we can all get better and you will see higher quality outcomes vs. using language that leads to community members shutting down. Once we feel like we are on the defense, we all become less effective at solving problems. I don't think we want people submitting PRs or responding to issue out of a place of anxiety/defensiveness. 

I believe we all want this community to be a battle ground of ideas so we can go far together without worrying about if our point of view is correct. If we don't,  my concern is we miss valuable insights that haunt us further down the road or great contributions because users fear interacting with our community. I know I had anxiety jumping into what is seemingly such a tight knit community around such a complicated project. In the end, we need more brains to grow and providing a safe idea farm benefits all distributions and users a like. 

Some good material on these topics:
* Never Split the Difference: Negotiating as if Your Life Depended on It
* Nonviolent Communication: A Language of Life: Life-Changing Tools for Healthy Relationship
* Ego, Authority, Failure: Using Emotional Intelligence like a Hostage Negotiator to Succeed as a Leader
* Building Brand Communities: How Organizations Succeed by Creating Belonging 

Hopefully this feedback was valuable and we can get some better outcomes by adjusting our communication strategies :-)
